data_diff/dbt.py

```diff
@@ -220,16 +222,28 @@ def _local_diff(diff_vars: DiffVars) -> None:
 
 
 def _cloud_diff(diff_vars: DiffVars) -> None:
+    datafold_host = os.environ.get("DATAFOLD_HOST")
+    if datafold_host is None:
+        datafold_host = "https://app.datafold.com"
+    datafold_host = datafold_host.rstrip("/")
+    rich.print(f"Cloud datafold host: {datafold_host}")
+
     api_key = os.environ.get("DATAFOLD_API_KEY")
+    if not api_key:
+        rich.print("[red]API key not found, add it as an environment variable called DATAFOLD_API_KEY.")
```

Comment:
This might be inconvenient to set the env var in every terminal tab. Putting the token in the standard shell setup scripts might (and will) compromise the security.

It is better to switch to using [`keyring`](https://pypi.org/project/keyring/) sooner (maybe not in this PR, but fast enough).

Reply:
I think we should support env vars too by default because it is a common wide-spreaded way.
I am planning to add [keyring](https://pypi.org/project/keyring/) in the next iteration although i have read that it might be some problem to install in on Linux. Anyway, i will try.